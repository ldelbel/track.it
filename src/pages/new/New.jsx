import React from 'react'
import PropTypes from 'prop-types'

const New = props => {
  return (
    <div>
      
    </div>
  )
}

New.propTypes = {

}

export default New
