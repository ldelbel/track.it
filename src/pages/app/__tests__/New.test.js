import React from 'react';
import renderer from 'react-test-renderer';
import New from '../new';

describe('new component', () => {
  it('renders correctly', () => {
    const newc = renderer.create(<New />).toJSON();
    expect(newc).toMatchSnapshot();
  });
});
