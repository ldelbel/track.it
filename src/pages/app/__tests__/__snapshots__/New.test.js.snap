// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new component renders correctly 1`] = `
<main
  className="bg"
>
  <div
    className="container"
  >
    <img
      alt="run"
      className="img"
      src="running.png"
    />
    <button
      className="button"
      onClick={[Function]}
    >
      New Running Session
    </button>
  </div>
</main>
`;
